nni/algorithms/compression/v2/pytorch/pruning/tools/sparsity_allocator.py

```diff
@@ -141,7 +141,7 @@ def generate_sparsity(self, metrics: Dict) -> Dict[str, Dict[str, Tensor]]:
             group_mask = torch.cat(group_mask, dim=0)
 
             for name, metric in group_metric_dict.items():
-                metric = (metric - metric.min()) * group_mask
+                metric = metric * group_mask
```

Comment:
why the previous version minus `metric.min()`?

Reply:
Because we want to support the metric with negative values before, but I found this is not a good way, so remove it. We don't have a negative metric except the `movement pruning score` right now.